cmd/mc-router/main.go

```diff
@@ -43,6 +43,8 @@ type Config struct {
 	MetricsBackend       string   `default:"discard" usage:"Backend to use for metrics exposure/publishing: discard,expvar,influxdb"`
 	UseProxyProtocol     bool     `default:"false" usage:"Send PROXY protocol to backend servers"`
 	MetricsBackendConfig MetricsBackendConfig
+
+	SimplifySRV bool `default:"false" usage:"Enable cleaning SRV records as they break parsing"`
```

Comment:
Looking at this again, I don't like the wording of the parameter description. I would prefer something like:
```suggestion
	SimplifySRV bool `default:"false" usage:"Simplify fully qualified SRV records for mapping"`
```